hardware/rtc.py

```diff
@@ -25,16 +25,17 @@
 
 
 def get_rtc():
-    cmd = ['timedatectl', 'status', '--no-pager']
-    stdout = check_output(cmd).decode("utf-8")
-    if stdout:
-        match = re.search(r'RTC in local TZ: ([a-z]+)$', stdout)
-        if match:
-            LOG.info('Is RTC set to UTC: %s' % match.group(1))
-            return match.group(1)
-
-        LOG.warning('Unable to determine RTC timezone (no match)')
-    else:
-        LOG.warning('Unable to determine RTC timezone (no output)')
-
+    try:
+        cmd = ['timedatectl', 'status', '--no-pager']
+        stdout = check_output(cmd).decode("utf-8")
+        if stdout:
+            match = re.search(r'RTC in local TZ: ([a-z]+)$', stdout)
+            if match:
+                LOG.info('Is RTC set to UTC: %s' % match.group(1))
+                return match.group(1)
+            LOG.warning('Unable to determine RTC timezone (no match)')
+        else:
+            LOG.warning('Unable to determine RTC timezone (no output)')
+    except FileNotFoundError:
```

Comment:
Let's limit the try..except block to the command that can raise it?

Reply:
Also isn't `FileNotFoundError` a Python 3 thing?

Reply:
1) Thanks, changed.
2) Good pickup, using OSError now which is the python2 equivalent and the parent exception to FileNotFoundError